shotgun_api3/lib/mockgun/mockgun.py

```diff
@@ -244,14 +244,23 @@ def schema_field_read(self, entity_type, field_name=None):
 
 
     def find(self, entity_type, filters, fields=None, order=None, filter_operator=None, limit=0, retired_only=False, page=0):
-        
-
         self.finds += 1
 
         self._validate_entity_type(entity_type)
         # do not validate custom fields - this makes it hard to mock up a field quickly
         #self._validate_entity_fields(entity_type, fields)
 
+        # Configure fields
+        if fields is None:
+            fields = set(["type", "id"])
+        else:
+            fields = set(fields) | set(["type", "id"])
+
+        # Include fields from the order argument in the searched fields
+        if order:
+            for o in order:
```

Comment:
Your are right, the fields were leakings! I've pushed a new commit that fix this.